internal/engine/postgresql/contrib/pgcrypto.go

```diff
@@ -104,7 +104,7 @@ func Pgcrypto() *catalog.Schema {
 			Name: "digest",
 			Args: []*catalog.Argument{
 				{
-					Type: &ast.TypeName{Name: "text"},
+					Type: &ast.TypeName{Name: "bytea"},
```

Comment:
Changed the order of these, as they affected the end-to-end tests, and just wanted to preserve legacy behavior.